hardhat.config.cjs

```diff
@@ -0,0 +1,6 @@
+require("@nomicfoundation/hardhat-toolbox");
+
+/** @type import('hardhat/config').HardhatUserConfig */
+module.exports = {
+  solidity: "0.8.24",
```

Comment:
should we add the same version to foundry.toml?

Reply:
actually thanks for leaving this comment - I think we can get rid of hardhat altogether, I don't think it makes sense to have it in this package; hardhat would be more useful for users of this npm package